Vagrantfile

```diff
@@ -0,0 +1,24 @@
+# -*- mode: ruby -*-
+# vi: set ft=ruby :
+
+# Vagrantfile API/syntax version. Don't touch unless you know what you're doing!
+VAGRANTFILE_API_VERSION = "2"
+
+Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
+  config.vm.box = "ubuntu/trusty64"
+  #config.vm.box_url = "https://atlas.hashicorp.com/ubuntu/trusty64"
+  config.vm.box_url = "https://atlas.hashicorp.com/ubuntu/boxes/trusty64/versions/20150928.0.0/providers/virtualbox.box"
+  config.vm.network "forwarded_port", guest: 80, host: 8080
```

Comment:
no that was just for my testing, I think the port is automatically forwarded for you anyway. it doesn't really work properly because of the way ckan redirects anwya